static/app/views/issueDetails/streamline/sidebar/seerSection.tsx

```diff
@@ -99,6 +99,11 @@ export default function SeerSection({
 
   const aiConfig = useAiConfig(group, project);
   const issueTypeConfig = getConfigForIssueType(group, project);
+
+  if (!aiConfig.areAiFeaturesAllowed) {
+    return null;
+  }
+
```

Comment:
@jennmueng I know this is still in draft, but the `Resources` section below is not AI dependent and should still be displayed in the EU. This is an artifact of this section previously being called the 'Solutions Hub' and not leaning toward AI only, so it might be worth pulling that out into a separate component that can render independently.

Reply:
Just added a commit that shows it!